repository-structure.md

```diff
@@ -0,0 +1,75 @@
+# Common Repository Structure
+
+Hyperledger projects are required to maintain a standard set of files in each repository. This
+document describes the required and recommended files.
+
+## Required Files with Specified Content
+
+Repositories MUST have these files with the specific content in the linked files, or a file with a
+link to the specified content with minimal exposition. These files MUST be at the root of the
+repository.
+
+-   [`LICENSE`](https://www.apache.org/licenses/LICENSE-2.0.txt) - https://www.apache.org/licenses/LICENSE-2.0.txt\
+    (Unless an exception has been made by the Hyperledger Governing Board)
+-   [`CODE_OF_CONDUCT.md`](https://wiki.hyperledger.org/community/hyperledger-project-code-of-conduct) - https://wiki.hyperledger.org/community/hyperledger-project-code-of-conduct
```

Comment:
Should we be pointing to https://tsc.hyperledger.org/code-of-conduct.html as the official document?

Reply:
Done. I take it this will be the new cannonical home of the CoC doc?

Reply:
That was my impression -- that we were moving all TSC governing process documents to GitHub.